### DISCUSSION TITLE
a/electron-regedit

### DISCUSSION THREAD

**Discussion by giscus** [sincerefly/giscus-data#4] [Announcements]
# a/electron-regedit

https://blog.yasking.org/a/electron-regedit.html

**Comment by N5fd2de** [comment]
https://atom.io/download/electron请问博主，这个怎么访问啊

**Reply by sincerefly** [reply]
看起来 node-gyp 的下载地址有了调整

例如：原路径为 https://atom.io/download/electron/v19.0.0/node-v19.0.0-headers.tar.gz
新的路径是 https://electronjs.org/headers/v19.0.0/node-v19.0.0-headers.tar.gz

---

以上是在 https://github.com/atom/atom/issues/25481 看到的。

建议

1. 不指定 --dist-url 尝试下载，报错后可能会发现新版的下载路径。
2. 使用全局代理，让脚本自动下载。
3. 这篇文章很老了，可能你遇到的问题已经有了更好的解决方式，最好尝试新的思路和方法。
